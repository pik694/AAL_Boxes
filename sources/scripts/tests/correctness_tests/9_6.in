9
34 31 50
32 38 53
60 76 24
38 58 8
14 80 41
36 38 84
49 65 49
57 85 37
51 18 59

