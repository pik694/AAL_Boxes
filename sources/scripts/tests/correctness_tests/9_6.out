3 4 0 1 6 8 2 5 7 
794544->490690
